packages/react/src/chip/Chip.css.ts

```diff
@@ -0,0 +1,117 @@
+import { theme } from "../styles";
+import { createVar, style } from "../vanilla-extract";
+import { type RecipeVariants, recipe } from "../vanilla-extract";
+
+const accentColorVar = createVar();
+const solidAccentColorVar = createVar();
+const subtleAccentColorVar = createVar();
+
+export const chip = recipe({
+  base: [
+    {
+      alignItems: "center",
+      display: "inline-flex",
+      flexDirection: "row",
+      gap: "4",
+      justifyContent: "center",
+      overflow: "hidden",
+      rounded: "md",
+      transition: "colors",
+    },
+    style({
+      position: "relative",
+      userSelect: "none",
+
+      selectors: {
+        "&:focus-visible": {
+          outlineOffset: "1px",
+          outlineStyle: "solid",
+          outlineWidth: "2px",
```

Comment:
Dont we need color? 

```suggestion
          outline: "2px solid",
          outlineOffset: "1px",          
```

Reply:
it was being set in the ColorScheme part.